examples/asr/emformer_rnnt/tedlium3/eval_pipeline.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
The official way of writing shebang line for Python3 is `python3` for the time before Python2 was deprecated. I cannot find the exact one that states it but the relevant PEP is https://www.python.org/dev/peps/pep-0394/

```
#!/usr/bin/env python3
```

Reply:
addressed in https://github.com/pytorch/audio/pull/2222/commits/f3d19ac9ee661750a074d1e406a5f8609dddf796